src/content/en/updates/2017/09/chrome-62-deprecations.md

```diff
@@ -2,9 +2,10 @@ project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
 description: A round up of the deprecations and removals in Chrome 62 to help you plan. In this version, security improvements, further webkit deprecations, and more.
 
-{# wf_updated_on: 2017-10-24 #}
+{# wf_updated_on: 2017-12-15 #}
 {# wf_published_on: 2017-09-15 #}
 {# wf_tags: deprecations,removals,chrome62 #}
+{# wf_blink_components: Blink #}
```

Comment:
Can we tag this with all relevant component names for this update? The generic Blink tag isn't that useful.

Reply:
Are they comma separated?

Reply:
yep. blinkcomponents-b48b5.firebaseapp.com can help you generate it.

Reply:
Done.